app/parse.py

```
@@ -8,8 +17,53 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def get_page(page_number: int) -> bytes:
+    url = urljoin(BASE_URL, f"page/{page_number}/")
+    try:
+        response = requests.get(url, timeout=5)
+        response.raise_for_status()
+        return response.content
+    except RequestException as e:
+        print(f"Request failed for {url}: {e}")
+        return b""
```

get_page swallows RequestException and returns `b""`. Returning empty bytes causes get_all_quotes to stop quietly and hides network issues. Instead either:
- implement a small retry mechanism for transient errors and then raise if all retries fail, or
- raise a clear exception (e.g., re-raise the RequestException) so callers can fail loudly.
At minimum, returning None or raising will make the failure explicit and avoid silent test failures (requirement 3.8).